test/libyul/YulOptimizerTestCommon.cpp

```diff
@@ -89,8 +89,7 @@ YulOptimizerTestCommon::YulOptimizerTestCommon(
 			disambiguate();
 			NameDisplacer{
 				*m_nameDispenser,
-				{"illegal1"_yulstring, "illegal2"_yulstring, "illegal3"_yulstring, "illegal4"_yulstring, "illegal5"_yulstring}
-			}(*m_ast);
+				{"illegal1"_yulname, "illegal2"_yulname, "illegal3"_yulname, "illegal4"_yulname, "illegal5"_yulname}}(*m_ast);
```

Comment:
Now it looks as if you were passing `(*m_ast)` to a call on what's inside the brace but it's really a call on `NameDisplacer. The original formatting was correct:

```suggestion
				{"illegal1"_yulname, "illegal2"_yulname, "illegal3"_yulname, "illegal4"_yulname, "illegal5"_yulname}
			}(*m_ast);
```